musicplatform/albums/models.py

```diff
@@ -7,6 +7,6 @@ class Album(models.Model):
     release_datetime = models.DateTimeField(blank=False)
     cost = models.DecimalField(max_digits=21, decimal_places=2)
     artist = models.ForeignKey(Artist, on_delete=models.CASCADE)
-
+    isApproved=models.BooleanField(default=False,help_text='Approve the album if its name is not explicit')
```

Comment:
it should be is_approved, as this is the convention naming for the django model fields